### ISSUE TITLE
Dropdown component has redundant labels

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#2332]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: all

#### Describe the issue:

Accessibility: The button inside the Dropdown component (list) has redundant labels/instructions.

Example values:
Aria-label (button): "listItem" 
Button text: "listItem" 
Title (button): "listItem" 

Example snippet (DOM):
`<button type="button" id="Dropdown7-list6" data-index="6" data-is-focusable="true" class="ms-Dropdown-item item_5ad99a4c ms-Button ms-Button--command css-3d3mgq" role="option" aria-selected="false" title="listItem" aria-label="listItem" tabindex="-1">
    <div class="ms-Button-flexContainer css-130zv93">
        <span class="ms-Dropdown-optionText optionText_5ad99a4c">listItem</span>
    </div>
</button>`

#### Expected behavior:

If the title and aria-label just copies the button text, they are redundant and unnecessary, and should be removed.

**Comment by betrue-final-final** [comment]
This is correct. We shouldn't be using the ARIA label to name the type of control if it's a semantic html control like a button or radio.
